include/crocoddyl/multibody/costs/contact-cop-position.hpp

```diff
@@ -127,10 +136,12 @@ struct CostDataContactCoPPositionTpl : public CostDataAbstractTpl<_Scalar> {
 
   pinocchio::DataTpl<Scalar>* pinocchio;
   MatrixXs Arr_Ru;
-  boost::shared_ptr<ContactDataAbstractTpl<Scalar> > contact; //!< spatial force expressed in world coordinates
-  pinocchio::SE3Tpl<Scalar> fiMo; //!< SE3 object with origin at contact point and rotation aligned with the world frame
-  pinocchio::ForceTpl<Scalar> f; //!< cartesian force expressed in world coordinates 
-  Vector3s cop; //!< center of pressure
+  boost::shared_ptr<ContactDataAbstractTpl<Scalar> > contact; //!< contact force
+  pinocchio::ForceTpl<Scalar> f; //!< transformed contact force
+  std::size_t frame_id; //!< contact frame id
```

Comment:
Do not store frame_id, cop_region and normal in the data.
These objects belong to the model.